tests/NClient.Packages/NClient.Packages.Tests/ToolTest.cs

```diff
@@ -0,0 +1,63 @@
+﻿#if NET5_0
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Threading;
+using System.Threading.Tasks;
+using FluentAssertions;
+using FluentAssertions.Extensions;
+using NClient.Packages.Tests.Helpers;
+using NUnit.Framework;
+
+namespace NClient.Packages.Tests
+{
+    public class ToolTest
+    {
+        [Test]
+        public async Task Install()
```

Comment:
It's really working now? We pass "dotnet .... " as part of command and use "dotnet" as filename. It looks like double dotnet using. Process output is correct? Can we simplify code and use "dotnet" only in FileName?

Reply:
It works, but the second "dotnet" command is redundant. I forgot to delete it. Done